css/style.css

```diff
@@ -291,4 +291,63 @@ html {
   height: 55rem;
   border-radius: 100%;
   box-shadow: 0 3rem 5rem rgb(0, 0, 0, 0.3);
+}
+
+/******************************/
+/* INGREDIENTS SECTION */
+/******************************/
+
+.ingredients-section {
+  background: url('../gallery/banners/ingredients-banner.webp');
+  background: url('../gallery/banners/ingredients-banner.jpg');
+  background-size: cover;
+  width: 100%;
+  padding: 10rem 0;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.dark-square {
```

Comment:
What if tomorrow we change background color of this block?
It's better to name it for example `ingredients-figure-container`.

Reply:
Ok